### PR TITLE
maracluster tool added. 

### PR DESCRIPTION
**Pull request by ypriverol**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [x] LABEL about.documentation   
  - [x] LABEL about.license_file
  - [x] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

File: maracluster/1.02.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+################## MAINTAINER ######################
+MAINTAINER Yasset Perez-Riverol <ypriverol@gmail.com>
+
+################## INSTALLATION ######################
+
+ENV DEBIAN_FRONTEND noninteractive
+
+## Update and install packages
+RUN apt-get update -y && apt-get install -y g++ build-essential cmake zlib1g-dev libbz2-dev libboost-all-dev wget
```
Comment (osallou) [inline maracluster/1.02.1/Dockerfile]: Why compiler, cmake etc....  as you install a .deb   non should be necessary except wget

File: maracluster/1.02.1/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+################## BASE IMAGE ######################
+FROM ubuntu:latest
```
Comment (osallou) [inline maracluster/1.02.1/Dockerfile]: Should specify an ububtu version for reproducibility

**Review by osallou — CHANGES_REQUESTED** [review]

File: maracluster/1.02.1/Dockerfile (hunk 2 above)
Comment (ypriverol) [reply maracluster/1.02.1/Dockerfile]: Which version are we using now ?

Comment (osallou) [reply maracluster/1.02.1/Dockerfile]: Don t care which, just need to specify one, and if using a deb, should rather use a version matching this deb else may not work according to related shared libs

**Comment by ypriverol** [comment]
Hi @osallou I did the changes you requested: 
- [x] ubuntu version specified. 
- [x] removed the dependencies not needed to work the tool. 
- [x] The tool has been tested locally.

File: maracluster/1.02.1/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+################## MAINTAINER ######################
+MAINTAINER Yasset Perez-Riverol <ypriverol@gmail.com>
+
+################## INSTALLATION ######################
+
+ENV DEBIAN_FRONTEND noninteractive
+
+## Update and install packages
+RUN apt-get update -y && apt-get install -y g++ build-essential zlib1g-dev libbz2-dev libboost-all-dev wget
```
Comment (osallou) [inline maracluster/1.02.1/Dockerfile]: Why g++ build etc... just need wget to get the deb files. Others are not needed and will increase container size 

Other expected libs should be anyway installed automatically alobg deb file

**Review by osallou — CHANGES_REQUESTED** [review]

File: maracluster/1.02.1/Dockerfile (hunk 3 above)
Comment (ypriverol) [reply maracluster/1.02.1/Dockerfile]: @osallou if I leave the `wget` only in local tests it doesn't work. I guess the deb is not installing the dependencies?

Comment (ypriverol) [reply maracluster/1.02.1/Dockerfile]: exact error: 
```
shared libraries: libgomp.so.1: cannot open shared object file: No such file or directory
```

Comment (ypriverol) [reply maracluster/1.02.1/Dockerfile]: Libraries removed and it is working. Only needs that remaining library.

**Pull request merged**